locales/templates/app.pot

```diff
@@ -70,13 +70,13 @@ msgctxt "text_selection_search_action"
 msgid "Search in Firefox Rocket"
```

Comment:
Is there a reason you're keep Firefox Rocket here but changing it below with %1$s?
Might be worth checking if there are other instances in strings still, maybe?

Reply:
It's decided in compile time (which means, we have no way to change it when it is shown). It's a current limitation.